### PR TITLE
mesa: Cherrypick !30562 to fix vaapi for AMD GPUs

### PR DESCRIPTION
**Pull request by 1Naim**

### CODE REVIEW

**Comment by 1Naim** [comment]
This should also be fixed in 24.2.1

File: mesa/mesa/PKGBUILD [hunk 1]
```diff
@@ -166,10 +168,12 @@ sha256sums=('c02bb72cea290f78b11895a0c95c7c92394f180d7ff66d4a762ec6950a58addf'
             '5443807d6dff69373d433ab9ef5378ad8df50ca6298caf15de6e52e24aaf54d5'
             '3c93a82e8d145725dcbaf44e5ea887c8a869efdcc28706df2d08c69e17077183'
             '692fcb63b64b1758029e0a96ee63e049ce8c5948587f2f7208df04625e5f6b56'
-            '901fa70d88b9d6c98022e23b4136f9f3e54e4662c3bc1bd1d84a42a9a0f0c1e9')
+            '901fa70d88b9d6c98022e23b4136f9f3e54e4662c3bc1bd1d84a42a9a0f0c1e9'
+            '2cb41add5ea8e31ea01b1f9ff447670c9afa3f0f934842d86e93f27f3972e7ac')
 
 prepare() {
   cd mesa-$pkgver
+  patch -Np1 < ../radeonsi-fix-efc-rework.patch
```
Comment (vnepogodin) [inline mesa/mesa/PKGBUILD:176]: can you put a comment above the patch

File: mesa/lib32-mesa/PKGBUILD [hunk 2]
```diff
@@ -152,10 +154,12 @@ sha256sums=('c02bb72cea290f78b11895a0c95c7c92394f180d7ff66d4a762ec6950a58addf'
             '5443807d6dff69373d433ab9ef5378ad8df50ca6298caf15de6e52e24aaf54d5'
             '3c93a82e8d145725dcbaf44e5ea887c8a869efdcc28706df2d08c69e17077183'
             '692fcb63b64b1758029e0a96ee63e049ce8c5948587f2f7208df04625e5f6b56'
-            '901fa70d88b9d6c98022e23b4136f9f3e54e4662c3bc1bd1d84a42a9a0f0c1e9')
+            '901fa70d88b9d6c98022e23b4136f9f3e54e4662c3bc1bd1d84a42a9a0f0c1e9'
+            '2cb41add5ea8e31ea01b1f9ff447670c9afa3f0f934842d86e93f27f3972e7ac')
 
 prepare() {
   cd mesa-$pkgver
+  patch -Np1 < ../radeonsi-fix-efc-rework.patch
```
Comment (vnepogodin) [inline mesa/lib32-mesa/PKGBUILD:162]: same here

File: mesa/mesa/PKGBUILD (hunk 1 above)
Comment (1Naim) [reply mesa/mesa/PKGBUILD:178]: Done, thanks!

**Review by vnepogodin — APPROVED** [review]
legit

**Pull request merged**